apps/rush-lib/src/cli/utilities/InstallManager.ts

```diff
@@ -602,13 +616,13 @@ export default class InstallManager {
     //       is optional or not, but it does not appear to do so. Also, this would result in strange behavior where
     //       people would have different node_modules based on their system.
 
-    const npmInstallArgs: string[] = ['install', '--no-optional'];
-    this.pushConfigurationNpmArgs(npmInstallArgs);
+    const installArgs: string[] = ['install', '--no-optional'];
```

Comment:
>--no-optional [](start = 47, length = 13)

Does pnpm support this?

Reply:
Is the above essay relevant to PNPM?

---
In reply to: [151787571](https://github.com/Microsoft/web-build-tools/pull/427#discussion_r151787571) [](ancestors = 151787571)

Reply:
No it is not, I will add a note about this.

---
In reply to: [151787839](https://github.com/Microsoft/web-build-tools/pull/427#discussion_r151787839) [](ancestors = 151787839,151787571)

Reply:
It does not appear to be supported by PNPM after a quick test

---
In reply to: [151787839](https://github.com/Microsoft/web-build-tools/pull/427#discussion_r151787839) [](ancestors = 151787839,151787571)